rpmconf/Install/zmsetup.pl

```diff
@@ -7225,6 +7262,8 @@ sub applyConfig {
 
   configInitMta();
 
+  configureLicenseDaemonService();
```

Comment:
inside this we are checking if service is enabled or not but service enablement status is set in ldap in configSetEnabledServices method in next line

Reply:
before configSetEnabledServices, all installed packages will be enabled. https://github.com/Zimbra/zm-build/blob/b1623ed42a66131fc2540363242cd9926eb143e0/rpmconf/Install/zmsetup.pl#LL668C33-L668C55